### PR TITLE
update hotspot settings ui

### PR DESCRIPTION
**Pull request by tyler-whitman**
Update hotspots settings ui to get ready for discovery mode and assert v2.

Simulator Screen Shot - iPhone 11 - 2021-03-16 at 12 16 16

### CODE REVIEW

File: src/features/hotspots/settings/HotspotSettings.tsx [hunk 1]
```diff
@@ -41,6 +41,10 @@ import { useHotspotSettingsContext } from './HotspotSettingsProvider'
 import Box from '../../../components/Box'
 import BackButton from '../../../components/BackButton'
 import animateTransition from '../../../utils/animateTransition'
+import BluetoothIcon from '../../../assets/images/bluetooth_icon.svg'
+import TransferIcon from '../../../assets/images/transfer_icon.svg'
+// import DiscoveryModeIcon from '../../../assets/images/discovery_mode_icon.svg'
```
Comment (matthewcarlreetz) [inline src/features/hotspots/settings/HotspotSettings.tsx:46]: remove?

File: src/features/hotspots/settings/HotspotSettings.tsx [hunk 2]
```diff
@@ -105,13 +109,14 @@ const HotspotSettings = ({ hotspot }: Props) => {
   }, [hotspot])
 
   const transferButtonTitle = useMemo(() => {
+    console.log(hasActiveTransfer)
```
Comment (matthewcarlreetz) [inline src/features/hotspots/settings/HotspotSettings.tsx:112]: remove?

**Review by matthewcarlreetz — APPROVED** [review]
There's one minor ui nitpick I'm seeing...
When the font size is set to huge there's an alignment issue with the Transfer Hotspot card
Simulator Screen Shot - iPhone SE (2nd generation) - 2021-03-16 at 16 24 32

Other than that it seems to work really well.

File: src/features/hotspots/settings/HotspotSettings.tsx (hunk 1 above)
Comment (tyler-whitman) [reply src/features/hotspots/settings/HotspotSettings.tsx:46]: I kept them for when we added those other code blocks back in

File: src/features/hotspots/settings/HotspotSettings.tsx (hunk 2 above)
Comment (tyler-whitman) [reply src/features/hotspots/settings/HotspotSettings.tsx:112]: Whoops yes this one ill remove

**Comment by tyler-whitman** [comment]
> There's one minor ui nitpick I'm seeing...
> When the font size is set to huge there's an alignment issue with the Transfer Hotspot card
> 
> Other than that it seems to work really well.

Good catch, I fixed it.

**Pull request merged**